### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by dikshadhote**
Following Features have been added 
1. Small text 
image
2. Grey text
image
3.Text alignment 
image

### CODE REVIEW

File: components/typography.css [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
 
- .typography-container{
-    margin: 2rem 0rem;
-    justify-items: center;
-    background-color: var(--component-secondary-color);
-    padding: 2rem;
-    border-radius: 10px;
- }
\ No newline at end of file
+.small-text {
```
Comment (Manju-sree) [inline components/typography.css:12]: please try to use utility class for these styles. this was suggested in most of the Review comments in first submission.

**Review by Manju-sree — APPROVED** [review]
just a small suggestion mentioned in comments, apart from that this PR looks fine to merge.

File: components/typography.css (hunk 1 above)
Comment (t007rushi) [reply components/typography.css:12]: I didn't get you, this isn't a utility class for text?
- Overall UI is Good 
- Just one thing reduce padding 
- Heading can be bold of larger font size for easy recognization

**Review by t007rushi — APPROVED** [review]

File: components/typography.css (hunk 1 above)
Comment (dikshadhote) [reply components/typography.css:12]: Its not utlity class its specific to typography for small text

**Pull request merged**